tests/functional/connect_another_device.js

```diff
@@ -101,16 +125,16 @@ define([
     'sign up Fx Desktop, verify different Fx Desktop': function () {
       // should sign in to sync here
       return this.remote
-        .then(openPage(SIGNUP_DESKTOP_URL, '#fxa-signup-header', {
+        .then(openPage(SIGNUP_DESKTOP_URL, SELECTOR_SIGNUP_HEADER, {
           forceUA: UA_STRINGS['desktop_firefox']
         }))
         .then(respondToWebChannelMessage('fxaccounts:can_link_account', { ok: true } ))
```

Comment:
Noticed you re-using strings `fxaccounts:can_link_account`, maybe pull it up for reuse?